core/src/main/resources/filodb-defaults.conf

```diff
@@ -40,13 +40,27 @@ filodb {
       # Eg: For copyTags `"_ns_" = [ "exporter", "job" ]`, when an incoming TimeSeries already has both `exporter`
       #     and `job`, then it will pickup only the value from `exporter` for `_ns_`.
       copyTags = {
-        "_ns_" = [ "_ns", "exporter", "job" ]
+        "_ns_" = ["_ns", "exporter", "job"]
       }
-      ignoreShardKeyColumnSuffixes = { "_metric_" = ["_bucket", "_count", "_sum"] }
+      ignoreShardKeyColumnSuffixes = {"_metric_" = ["_bucket", "_count", "_sum"]}
       ignoreTagsOnPartitionKeyHash = ["le"]
       metricColumn = "_metric_"
       # shard key columns should be in hierarchical order
       shardKeyColumns = ["_ws_", "_ns_", "_metric_"]
+
+      # Multi-column facets to be created on "partition-schema" columns.
+      # Eg: below config creates a facet "all-hosts-availZone" by concatenating
+      # the column values ("avail_zone", "rack", "host") "\u03C0" as delimiter
+      # If the data is hierarchical in nature, This feature lets us fetch the metadata very efficiently by using
+      # lucene facet feature.
+      #
+      # Note: only "string" column type supported.
+      # additionalFacets = [
```

Comment:
What's is ``additionalFacets`` vs ``multiColumnFacets``?

Just calling out, I believe this concatenation must be less than ``FACET_FIELD_MAX_LEN``

Reply:
ah good catch, i refactored later and forgot to update the example. will fix it

Reply:
`FACET_FIELD_MAX_LEN` this is being checked in `addFacet`. so field will be omitted if it crosses the threshold.